### ISSUE TITLE
Haskell error popup height too small

### ISSUE THREAD

**Issue by codygman** [flycheck/flycheck#1738]
This template is for **user support**.  If you are reporting an actual Flycheck bug, please use the relevant issue template instead.

## Checklist
- [x] I have searched existing issues and StackOverflow for solutions.
- [x] I have read the [Flycheck manual][].
- [x] I have read the [Troubleshooting Guide][].
- [x] I have checked my issue has not been reported yet.

## Problem description

Relevant bindings are very long in the haskell error popup and obscure the more useful typed holes.

 I want to work around this by making the maximum height for those popups larger.

- [x] I have read https://emacs.stackexchange.com/questions/28429/how-do-i-troubleshoot-emacs-problems

## System configuration
```
Syntax checkers for buffer CustomReport.hs in haskell-mode:

First checker to run:

  haskell-dante (explicitly selected)
    - may enable:    yes
    - predicate:     t
    - next checkers: haskell-hlint

Checkers that could run if selected:

  haskell-stack-ghc  select
    - may enable:    yes
    - executable:    Found at /home/cody/.nix-profile/bin/stack
    - next checkers: haskell-hlint

  haskell-ghc  select
    - may enable:    yes
    - executable:    Found at /home/cody/.nix-profile/bin/ghc
    - next checkers: haskell-hlint

Checkers that are compatible with this mode, but will not run until properly configured:

  haskell-hlint (disabled)
    - may enable:         Automatically disabled!
    - executable:         Not found
    - configuration file: Not found

Flycheck Mode is enabled. Use SPC u C-c ! x to enable disabled checkers.

--------------------

Flycheck version: 32snapshot
Emacs version:    28.0.50
System:           x86_64-pc-linux-gnu
Window system:    x

```

Emacs configuration:
- [ ] Plain Emacs / Custom configuration
- [ ] Spacemacs
- [x] Doom Emacs
- [ ] Other shared configuration

## Additional notes

[Flycheck manual]:  https://www.flycheck.org/en/latest/user/troubleshooting.html
[Troubleshooting guide]:  https://www.flycheck.org/en/latest/user/troubleshooting.html

**Comment by cpitclaudel** [comment]
Can you clarify what popups you're talking about, ideally with a screenshot? :)

**Comment by codygman** [comment]
> Can you clarify what popups you're talking about, ideally with a screenshot? :)

I will as soon as I can. I couldn't share the specific one because it was a private code base.

I tried making a minimal example but I'm not sure how to replicate the "related bindings" section being very large.

I tried finding a larger Haskell project that already has a shell.nix (to work with my setup) to see if one demonstrates the problem but couldn't find it.

I'll likely have to figure out one of those paths before I can post a screenshot sadly.

**Comment by cpitclaudel** [comment]
Just a blurred-out screenshots would be a great start; I don't even know what those popups are ^^

**Comment by cpitclaudel** [comment]
Hi there,

I'd really love to help, but I need a bit more info.  Any change we could get at least a screenshot, and a confirmation that the problem does happen when using only Flycheck?

**Comment by bbatsov** [comment]
Closing this as user support. The popup display height is controlled by Emacs's built-in display settings rather than Flycheck itself. You can customize variables like `max-mini-window-height` or use packages like `flycheck-popup-tip` or `flycheck-posframe` for more control over how errors are displayed. Hope that helps\!
